### DISCUSSION TITLE
Supported libraries list

### DISCUSSION THREAD

**Discussion by mackenza** [open-telemetry/opentelemetry-java-instrumentation#6816] [Q&A]
To be included on the list of supported libraries, does the client library need to be open source? If it's available in Maven and freely available but not open source, is it able to be included in the published list?

**Comment by trask** [comment]
hi @mackenza! I'd recommend registering external instrumentation in the [OpenTelemetry Registry](https://opentelemetry.io/registry) under Java Instrumentation
